src/main/kotlin/de/achimonline/ansible_lint/command/AnsibleLintCommandLine.kt

```diff
@@ -58,6 +59,7 @@ class AnsibleLintCommandLine(private val settings: AnsibleLintSettings = Ansible
 
         return GeneralCommandLine()
             .withEnvironment(System.getenv())
+            .withEnvironment("PATH", settings.path)
```

Comment:
This will completely override any existing PATH settings of the current system/environment settings.